src/features/counterfactual/FirstTxFlow.tsx

```diff
@@ -74,8 +74,7 @@ const FirstTxFlow = ({ open, onClose }: { open: boolean; onClose: () => void })
   }
 
   const onSwap = () => {
-    // TODO: Pre-filter DeFi category apps
-    router.push({ pathname: AppRoutes.apps.index, query: router.query })
+    router.push({ pathname: AppRoutes.apps.index, query: { ...router.query, categories: 'DEX' } })
```

Comment:
Could you add DEX to the SafeAppsTag in constants.ts?

Reply:
I think DEX is not the right tag (it only exists on staging). We should probably use DeFi or Aggregator. Afaiu adding it to the SafeAppsTags filters it out in `getCategoryOptions` which we need to check if its a valid tag and not some random string.